dask/dataframe/tests/test_groupby.py

```diff
@@ -2762,8 +2762,8 @@ def test_groupby_grouper_dispatch(key):
     pd_grouper = grouper_dispatch(pdf)(key=key)
     gd_grouper = grouper_dispatch(gdf)(key=key)
 
-    expect = pdf.groupby(pd_grouper).sum()
-    got = gdf.groupby(gd_grouper).sum()
+    expect = pdf.groupby(pd_grouper).sum(numeric_only=None)
+    got = gdf.groupby(gd_grouper).sum(numeric_only=None)
```

Comment:
Explicitly specified `numeric_only` here since the default value is getting deprecated by Pandas and it looks like that's introducing some failures; do pandas and cuDF have conflicting behavior for `numeric_only=None` in groupbys? cc @brandon-b-miller since it looks like you've done some work here around groupby dispatch

Reply:
My bad, looks like the failures were just a result of the fact that cuDF doesn't implement `numeric_only` yet, and defaults to the behavior of `numeric_only=True`.